doc/rtd/applications.rst

```diff
@@ -666,3 +666,78 @@ regular expression.
         --output_file ${out_dir}/${out_files}                   \
         --arrays PSL TMQ UBOT VBOT U850 V850
 
+Atmospheric Rivers Detection (ARs)
+----------------------------------
+TECA developed detection techniques to quantify the uncertainty of Atmospheric
+River events.
```

Comment:
This introduction doesn't introduce the material that follows very well. this should be revised.